packages/crash-handler/README.md

```diff
@@ -0,0 +1,98 @@
+
+# @dotcom-reliability-kit/crash-handler
+
+> **Warning**
+> This Reliability Kit package is **experimental** and should be used with caution in production applications. Please also let the Customer Products Reliability team know if you intend on using this.
+
+A method to bind an uncaught exception handler to ensure that fatal application errors are logged. It is a replacement for Sentry fatal error logging. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+  * [Usage](#usage)
+    * [`registerCrashHandler`](#registercrashhandler)
+    * [configuration options](#configuration-options)
+      * [`process`](#optionsprocess)
+  * [Compatibility](#compatibility)
+    * [Migrating from Sentry](#migrating-from-sentry)
+  * [Contributing](#contributing)
+  * [License](#license)
+
+
+## Usage
+
+Install `@dotcom-reliability-kit/crash-handler` as a dependency:
+
+```bash
+npm install --save @dotcom-reliability-kit/crash-handler
+```
+
+Include in your code:
+
+```js
+import registerCrashHandler from '@dotcom-reliability-kit/crash-handler';
+// or
+const registerCrashHandler = require('@dotcom-reliability-kit/crash-handler');
+```
+
+### `registerCrashHandler`
+
+The `registerCrashHandler` function can be used to bind an event handler to the [Node.js `process.uncaughtException` event](https://nodejs.org/api/process.html#event-uncaughtexception). This ensures that your application logs a final message before crashing in the event on an unexpected error or promise rejection. This function should only ever be called once in your app, normally alongside all your setup code (e.g. alongside creating an Express app).
```

Comment:
> normally alongside all your setup code (e.g. alongside creating an Express app)

With some of the other Reliability Kit packages, their ordering in the setup code is relevant - is that the case here (and is it worth saying so)?

Reply:
There _shouldn't_ be any ordering, but I guess we're more likely to catch errors if it's one of the first things that's called. Maybe I'll add a note that there's no explicit requirement but the earlier the better.